app/webpacker/components/Tickets/TicketWorkbenches/EditPersonTicketWorkbench.jsx

```diff
@@ -49,6 +79,9 @@ function EditPersonTicketWorkbenchForWrt({ ticketDetails, actingStakeholderId, s
       <EditPersonValidations
         ticketDetails={ticketDetails}
       />
+      <EditPersonRequestedChangesList
+        requestedChanges={ticket.metadata?.tickets_edit_person_fields}
```

Comment:
The `?` in here means you expect that the result can be `null` sometimes. But your component above (`EditPersonRequestedChangesList`) does not handle the case where its param (`requestedChanges`) indeed becomes null. It would just crash trying to map over the (potentially non-existent) list.